src/test/java/seedu/address/logic/commands/DeleteTagCommandTest.java

```diff
@@ -34,12 +37,12 @@ public void setUp() {
      */
     @Test
     public void execute_existingTag_success() {
+        // Create existing tag.
         Tag existingTag = BRIDES_SIDE;
 
-        List<Tag> existingTags = List.of(existingTag);
         model.addTag(existingTag);
 
-        DeleteTagCommand deleteTagCommand = new DeleteTagCommand(existingTags);
+        DeleteTagCommand deleteTagCommand = new DeleteTagCommand(List.of(existingTag), false);
```

Comment:
is there a reason for creating a list from existingTag instead of using existingTags?